api/server/router/container/container.go

```diff
@@ -61,6 +61,7 @@ func (r *containerRouter) initRoutes() {
 		router.NewPostRoute("/containers/{name:.*}/rename", r.postContainerRename),
 		router.NewPostRoute("/containers/{name:.*}/update", r.postContainerUpdate),
 		router.NewPostRoute("/containers/prune", r.postContainersPrune, router.WithCancel),
+		router.NewPostRoute("/commit", r.postCommit),
```

Comment:
Wait, why isn't this namespaced under `/containers/` ?

This also needs documentation changes in the swagger and API version history

Reply:
oh, ahem @vdemeester pointed me that this was already the case

would it be an idea to _add_ a new endpoint that _does_ use `/containers/` as a prefix?

Reply:
We could. I think it would be better to group it in with a bunch of API fixes. I'm not sure it's important enough for a one-off. There are a lot of API inconsistencies that we could fix at all once.